extensions/compiler/compiler.cmd.ts

```diff
@@ -18,10 +27,49 @@ export class CompileCmd implements Command {
   constructor(private compile: WorkspaceCompiler) {}
 
   async report([components]: [string[]], { verbose, noCache }: { verbose: boolean; noCache: boolean }) {
-    const compileResults = await this.compile.compileComponents(components, { verbose, noCache });
-    // eslint-disable-next-line no-console
-    console.log('compileResults', compileResults);
-    return `${compileResults.length} components have been compiled successfully`;
+    const startTimestamp = process.hrtime();
+    const spinner = ora('Compiling your components, hold tight.').start();
```

Comment:
@urigat , you can re-use the Ora spinner we use everywhere else. Please see the logger README here for explanations on how to use it. https://github.com/teambit/bit/tree/master/src/extensions/logger